recipe/meta.yaml

```diff
@@ -36,12 +36,16 @@ requirements:
     - python
     - conda-env >=2.6
     - enum34                 # [py<34]
-    - futures                # [py<34]
+    - futures >=3.0.0        # [py<34]
```

Comment:
I'm curious -- where does this constraint come from?

Reply:
I don’t remember exactly. But I’m pretty sure there was a reason. Guess I should have left a comment 🤦🏻‍♂️

Reply:
`git blame` to the rescue: https://github.com/conda/conda/pull/6855
If you want to add a comment, `# fix for https://bugs.python.org/issue20367` or the like should do.